### PR TITLE
Added Notebook class with Player pw check

### PR DESCRIPTION
**Pull request by rainSax**

### CODE REVIEW

**Comment by KevinJBenoit** [comment]
Nice! I was thinking of adding a grid system for "checkboxes" kinda like this:
https://www.codeproject.com/KB/game/ClueBoardGame/notebook.PNG
But we can do that later after we get the basics done.

**Comment by rainSax** [comment]
oh yes, that would be a nice formatting for the notebook!

File: Player.cpp [hunk 1]
```diff
@@ -55,6 +55,52 @@ void Player::showHand() {
 	std::cout << std::endl;
 }
 
+void Player::viewNotebook() {
+	std::string pw;
+	int choice;
+	std::cout << getName() << ", enter your password: ";
+	std::cin >> pw;
```
Comment (KevinJBenoit) [inline Player.cpp]: do we have to clear the buffer after this call to cin?

**Review by KevinJBenoit — COMMENTED** [review]
I'll check the Notebook over next week and then merge it into master! After that we can get started on creating turns

File: Player.cpp (hunk 1 above)
Comment (rainSax) [reply Player.cpp]: i've changed the cin >> pw command to getline(std::cin, pw) so it works with passwords with spaces. i don't think getline() leaves anything behind in the buffer but i'm not 100% sure.

**Pull request merged**